dask/array/image.py

```diff
@@ -36,17 +43,23 @@ def imread(filename):
     Dask array of all images stacked along the first dimension.  All images
     will be treated as individual chunks
     """
+    imread = imread or sk_imread
     filenames = sorted(glob(filename))
     if not filenames:
         raise ValueError("No files found under name %s" % filename)
 
     name = 'imread-%s' % tokenize(filenames, map(os.path.getmtime, filenames))
 
-    sample = sk_imread(filenames[0])
+    sample = imread(filenames[0])
```

Comment:
Because ``preprocess`` can change the image size, sample should be taken after it is applied?